test-framework/common/src/main/java/io/quarkus/test/common/TestHostLauncher.java

```diff
@@ -28,7 +28,7 @@ public void close() throws IOException {
 
     @Override
     public boolean listensOnSsl() {
-        return false;
+        return Boolean.getBoolean("quarkus.http.test-ssl-enabled");
```

Comment:
I personally very much dislike that `Boolean#getBoolean` implies reading from a system property. Can we please use 

```java
Boolean.parseBoolean(System.getProperty("quarkus.http.test-ssl-enabled", "false"))
```

?

Reply:
Sure thing